examples/example-gauth/src/main/java/io/grpc/examples/googleAuth/GoogleAuthClient.java

```diff
@@ -33,15 +33,19 @@
 
 /**
  * Example to illustrate use of Google credentials as described in
- * @see <a href="../../../../../../GOOGLE_AUTH_EXAMPLE.md">Google Auth Example README</a>
+ * @see <a href="../../../../../../README.md">Google Auth Example README</a>
  *
  * Also @see <a href="https://cloud.google.com/pubsub/docs/reference/rpc/">Google Cloud Pubsub via gRPC</a>
  */
 public class GoogleAuthClient {
+  /**
+   * GoogleAuthClient Logger.
```

Comment:
Remove the comments newly added for private members. It is not required and is not adding any value.

Reply:
Fixed